src/loaders/FileLoader.js

```diff
@@ -168,20 +168,21 @@ Object.assign( FileLoader.prototype, {
 
 			request.addEventListener( 'load', function ( event ) {
 
-				var response = this.response;
+				var target = event.target;
```

Comment:
I think the name `target` is a bit confusing. It would be more clear to highlight that `target` represents the `XMLHttpRequest` object in this context.